roles/ceph-purge/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: check if ceph is on the system already
+  stat: path=/var/lib/ceph/osd/
+  changed_when: false
+  register: ceph_on_system
+
+- name: clear osd devices
+  shell: dd if=/dev/zero of={{ item }} bs=1M count=512
```

Comment:
Unless shell redirection is used the command module should be used instead of shell.